components/Navbar/Navbar.tsx

```diff
@@ -3,47 +3,154 @@ import Link from "next/link";
 import styled from "styled-components";
 import device from "@styles/utils/breakpoints";
 import config from "config.json";
+import BurgerButton from "./BurgerButton/BurgerButton";
+import Image from "../../components/Image/Image";
+import LogoImage from "../../public/assets/icons/logo-orange.svg";
+import Adapt from "@components/UiKit/Adapt";
+import LinkAsButton from "@components/UiKit/LinkAsButton";
 
-const Nav = styled.nav<{ $open?: boolean }>`
-  background: ${({ theme }) => theme.orangeColor};
-  height: 60px;
+const Nav = styled.nav`
```

Comment:
This Navbar commponent is looking a bit crowded now.
I recomend moving styled-components into Navbar.styled.tsx (in the same folder)